storybook/components/CustomCanvas/CustomCanvas.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react"
+import { Source, SourceProps, Story, Unstyled } from "@storybook/blocks"
+import classnames from "classnames"
+import { v4 } from "uuid"
+import styles from "./CustomCanvas.module.scss"
+
+export type CustomCanvasProps = {
+  of: SourceProps["of"]
+  initialIsOpen?: boolean
+  className?: string
+}
+
+/**
+ * An almost-duplicate of Storybook's <Canvas>.
+ * Only use this when Canvas prevents your component rendering correctly (eg. using a popover).
+ */
+export const CustomCanvas = ({
```

Comment:
Thinking we just name this `Canvas` or if we're worried about being confused with Storybook's `KaizenCanvas`? While it is a custom implementation, I'm not sure if it's important to name it such if it can be used generally.

Reply:
Yeah I just wanted to give it a different name so it's easy to differentiate in the case you want to use both types in your docs somehow.

Happy to name it `KaizenCanvas` :)

Reply:
I guess better to leave the door open than close it!